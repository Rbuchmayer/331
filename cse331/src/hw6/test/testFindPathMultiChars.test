LoadGraph twoCharsOneBook twoCharsOneBook.tsv
LoadGraph twoCharsTwoBooksConnected twoCharsTwoBooksConnected.tsv
LoadGraph twoCharsTwoBooksNotConnected twoCharsTwoBooksNotConnected.tsv
LoadGraph complexGraph complexGraph.tsv

FindPath twoCharsOneBook A B
FindPath twoCharsOneBook B A
FindPath twoCharsOneBook A A
FindPath twoCharsOneBook B B
FindPath twoCharsOneBook B C

FindPath twoCharsTwoBooksConnected A B
FindPath twoCharsTwoBooksConnected B A
FindPath twoCharsTwoBooksConnected A A
FindPath twoCharsTwoBooksConnected B B
FindPath twoCharsTwoBooksConnected A C

FindPath twoCharsTwoBooksNotConnected A B
FindPath twoCharsTwoBooksNotConnected B A
FindPath twoCharsTwoBooksNotConnected A A
FindPath twoCharsTwoBooksNotConnected B B
FindPath twoCharsTwoBooksNotConnected C A

FindPath complexGraph A H
FindPath complexGraph G H
FindPath complexGraph H H
FindPath complexGraph A B
FindPath complexGraph B A
FindPath complexGraph C A
FindPath complexGraph B B
FindPath complexGraph B F
FindPath complexGraph G C
FindPath complexGraph M C