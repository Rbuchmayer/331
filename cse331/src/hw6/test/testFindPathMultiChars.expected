loaded graph twoCharsOneBook
loaded graph twoCharsTwoBooksConnected
loaded graph twoCharsTwoBooksNotConnected
loaded graph complexGraph

path from A to B:
A to B via 1
path from B to A:
B to A via 1
path from A to A:
path from B to B:
unknown character C

path from A to B:
A to B via 1
path from B to A:
B to A via 1
path from A to A:
path from B to B:
unknown character C

path from A to B:
no path found
path from B to A:
no path found
path from A to A:
path from B to B:
unknown character C

path from A to H:
no path found
path from G to H:
no path found
path from H to H:
path from A to B:
A to D via 1
D to B via 2
path from B to A:
B to D via 2
D to A via 1
path from C to A:
C to F via 3
F to A via 1
path from B to B:
path from B to F:
B to D via 2
D to F via 1
path from G to C:
G to C via 3
unknown character M